app/models/category.rb

```diff
@@ -1,2 +1,6 @@
 class Category < ApplicationRecord
+  has_many :words, :dependent => :destroy
```

Comment:
@Masa08 We will use the other format for constructing hashes, which is `{key: :value}`, instead of `{:key => :value}`

This is a standard that we need to follow. Please check other files also.